### ISSUE TITLE
Could not find com.klarna.mobile:sdk:2.1.8 (version 2.2.0 and 2.3.0)

### ISSUE THREAD

**Issue by Kevinrob** [datatrans/android-sdk#5]
With the version `2.3.0`, Gradle can't find `com.klarna.mobile:sdk:2.1.8`.

`implementation("ch.datatrans:android-sdk:2.3.0")`

error:
```
9: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:mergeDebugNativeLibs'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.klarna.mobile:sdk:2.1.8.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/klarna/mobile/sdk/2.1.8/sdk-2.1.8.pom
       - https://repo.maven.apache.org/maven2/com/klarna/mobile/sdk/2.1.8/sdk-2.1.8.pom
       - https://jitpack.io/com/klarna/mobile/sdk/2.1.8/sdk-2.1.8.pom
       - https://datatrans.jfrog.io/artifactory/mobile-sdk/com/klarna/mobile/sdk/2.1.8/sdk-2.1.8.pom
     Required by:
         project :app > ch.datatrans:android-sdk:2.3.0
```

**Comment by Charchad** [comment]
I see the same error

**Comment by richiehug** [comment]
Hey @Kevinrob / @Charchad - sorry for the delay. On it. Will be back with an answer asap.

**Comment by bubbabuford** [comment]
I am also dealing with the same problem.

**Comment by richiehug** [comment]
Hey @Kevinrob / @Charchad / @bubbabuford - we’ll release a bugfix to fix this issue. Starting this next release, Klarna won’t be included by our SDK by default. So this error will be gone. If you are using Klarna going forward, you’ll need to include the Klarna repository below in build.gradle. Including this extra repository is the fix to build properly in 2.2.0 and 2.3.0.

```
maven { 
   url 'https://x.klarnacdn.net/mobile-sdk/' 
}
```

**Comment by richiehug** [comment]
Fixed in 2.3.1 ([link](https://github.com/datatrans/android-sdk/releases/tag/2.3.1)) 👨🏻‍💻.
